### ISSUE TITLE
Aquarius spitting out nonstop ERRORs in ocean.py unit tests

### ISSUE THREAD

**Issue by trentmc** [oceanprotocol/aquarius#414]
**Summary**

If I run ocean.py unit tests (with barge), the log of the console running barge console gets filled with the following:
```console
aquarius_1         | 2021-03-16 14:13:27,523 - aquarius - ERROR - get_ddo: NotFoundError(404, '{"_index":"aquarius","_type":"_doc","_id":"did:op:8761f7322680D916fe081Acafb157B1ea5a1c743","found":false}')
aquarius_1         | 2021-03-16 14:13:27 2ce1d6ca440c root[50] ERROR Dao.get -- asset with id did:op:8761f7322680D916fe081Acafb157B1ea5a1c743 was not found, original error was:NotFoundError(404, '{"_index":"aquarius","_type":"_doc","_id":"did:op:8761f7322680D916fe081Acafb157B1ea5a1c743","found":false}')
aquarius_1         | 2021-03-16 14:13:27,740 - aquarius - ERROR - get_ddo: NotFoundError(404, 
...
```

**To reproduce**

Go through ocean.py's [developers.md](https://github.com/oceanprotocol/ocean.py/blob/main/READMEs/developers.md) up to the point of running unit tests. In the tests you will see the above issue.

To avoid running all the tests, even just running this will spit logs like the above:
```console
pytest ocean_lib/assets/test/test_ocean_assets.py
```

**Comment by trentmc** [comment]
Marking as a bug given that it spits out nonstop ERROR message, which means it's flagged an actual bug, or it's way too zealous in flagging things and it's a DX bug.

**Comment by alexcos20** [comment]
not sure that this is a bug.
There is a resolve call with a specific DID, which is not found.  We need to track and see who is trying to resolve that asset and why it's not there

**Comment by DMats** [comment]
Just to add another data point to this issue, two of the `test_ocean_assets.py` tests fail.

```
FAILED ocean_lib/ocean/test/test_ocean_assets.py::test_register_asset - Exception: Unable to search for DDO: b'<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">\n<title>500 Internal Server Error</title>\n<h1>Internal Ser...
FAILED ocean_lib/ocean/test/test_ocean_assets.py::test_ocean_assets_search - Exception: Unable to search for DDO: b'<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">\n<title>500 Internal Server Error</title>\n<h1>Interna...
```

**Comment by DMats** [comment]
Possibly related https://discordapp.com/channels/612953348487905282/720631837122363412/824193052289269760
>greybandit — Today at 4:08 AM
Hi! I was using the above API and similar APIs like that to fetch data. Now they are not working. Can anyone point out the problem and help me fix it?
![image.png](https://images.zenhubusercontent.com/604f6e0302390d3e079501ce/b92590b5-e5e5-4a03-83bc-adb7492182ac)

**Comment by calina-c** [comment]
@DMats regarding your first comment, Trent has opened another issue #255 (already fixed in Aqua 2.2.6 but need to update barge, I think)

Regarding your second comment, "nativeSearch" is now enabled by default, and the top "query" key should only contain native ES queries. Removing `"nativeSearch": 1` does the trick.

I'm working on the logging level for assets not found.

**Comment by DMats** [comment]
@calina-c Sounds good!  Thanks.
